src/server.ts

```diff
@@ -92,7 +92,7 @@ export async function buildServer (ctx: ViteBuildContext) {
     .replace('{{ APP }}', '<div id="__nuxt">{{ APP }}</div>')
     .replace(
       '</body>',
-      '<script type="module" src="/@vite/client"></script><script type="module" src="/client.js"></script></body>'
+      '<script type="module" src="/@vite/client"></script><script type="module" src="/_nuxt/client.js"></script></body>'
```

Comment:
Note for the future: We need to support router base and custom build dir (`nuxt.options.app.assetsPath` instead of hardcoding `/_nuxt)

Reply:
Related https://github.com/nuxt/vite/issues/166